apps/makescene/makescene.cc

```diff
@@ -766,6 +766,101 @@ find_max_scene_id (std::string const& view_path)
 
 /* ---------------------------------------------------------------- */
 
+/* Find corresponding cam file to file i in sorted directory. */
+bool
+find_corresponding_cam_file(util::fs::Directory const & dir, std::size_t i,
```

Comment:
Why is this iterating over other files and not just replacing the extension with `.cam` and checking if the file is there?